libs/types-common/src/Data/MessageQueue.hs

```diff
@@ -0,0 +1,21 @@
+module Data.MessageQueue
+  ( MessageQueueSettings (..),
+  )
+where
+
+import Data.Aeson (FromJSON)
+import Data.Text (Text)
+import GHC.Generics (Generic)
+import Prelude (Show, String)
+
+-- | Options for connecting to the message queue system
+data MessageQueueSettings = MessageQueueSettings
```

Comment:
come to think of it: this is not used for anything yet, right?  i'd prefer to not add new dead code, we already have a lot of that.  i've removed this; here is the commit for when you need to re-add it in the future: c0df89aafa7064484c571ae406dce1dc36747cbb